frontend/src/components/Blog/Story.tsx

```diff
@@ -50,6 +58,7 @@ const Story = () => {
         <div className="py-4">
           <ReactQuill value={blog?.content} readOnly={true} theme={'bubble'} />
         </div>
+        <ChatModule blogContent={blog?.content || ''} blogTitle={blog?.title} userId={user?.id || null} blogId={id || ''} />
```

Comment:
There is no need to pass these props to ChatModule. 

We can simply call useBlog inside ChatModule to fetch blog data
And userId from localStorage